src/app/pages/courses-page/actions-section/actions-section.component.html

```diff
@@ -1 +1,14 @@
-<p>actions-section works!</p>
+<div class="actions-section-container">
+  <div class="search-course-container">
+    <div class="form-group">
+      <input type="text" placeholder="Type to search" class="form-control" [(ngModel)]="searchString">
+    </div>
+    <button class="button button--large button--green" (click)="searchCourseHandler()">Search</button>
+  </div>
+  <div class="add-course-container">
+    <button class="button button--large button--blue">
+      <fa-icon [icon]="iconPlus"></fa-icon>
+      <span>Add course</span>
+    </button>
```

Comment:
you can even create a re-usable `AppButton` component.

Reply:
I think that in this case it will be enough to use css classes for different shape and color of buttons